docs/cni/kubernetes/GCE.md

```diff
@@ -0,0 +1,159 @@
+<!--- end of master only -->
+
+# Deploying Calico and Kubernetes on GCE 
+
+These instructions allow you to set up a Kubernetes v1.1.3 cluster with [Calico networking][calico-networking] on GCE using the [Calico CNI plugin][calico-cni]. This guide does not setup TLS between Kubernetes components.
+
+## 1. Getting started with GCE
+These instructions describe how to set up two CoreOS hosts on GCE.  For more general background, see 
+[the CoreOS on GCE documentation][coreos-gce].
+
+### 1.1 Install the gcloud tool
+If you already have the `gcloud` utility installed, and a GCE project configured, you may skip this step.
+
+Download and install GCE, then restart your terminal: 
+```
+curl https://sdk.cloud.google.com | bash
+```
+For more information, see Google's [gcloud install instructions][gcloud-instructions].
+
+Log into your account:
+```
+gcloud auth login
+```
+
+In the GCE web console, create a project and enable the Compute Engine API.  
+Set the project as the default for gcloud:
+```
+gcloud config set project PROJECT_ID
+```
+And set a default zone
+```
+gcloud config set compute/zone us-central1-a
+```
+
+### 1.2 Setting up GCE networking
+GCE blocks traffic between hosts by default; run the following command to allow Calico traffic to flow between 
+containers on different hosts:
+```
+gcloud compute firewall-rules create calico-ipip --allow 4 --network "default" --source-ranges "10.240.0.0/16"
+```
+You can verify the rule with this command:
+```
+gcloud compute firewall-rules list
+```
+
+<!--- master only -->
+### 1.3 Clone this project
+
+    git clone https://github.com/projectcalico/calico-containers.git
+<!--- else
+### 1.3 Clone this project, and checkout the **release** release
+
+    git clone https://github.com/projectcalico/calico-containers.git
+    git checkout tags/**release**
+<!--- end of master only -->
+    
+## 2. Deploy the VMs 
+Change into the directory for this guide.
+```
+cd calico-containers/docs/cni/kubernetes/
+```
+
+Deploy the Kubernetes master node using the following command:
+```
+gcloud compute instances create \
+  kubernetes-master \
+  --image-project coreos-cloud \
+  --image coreos-alpha-709-0-0-v20150611 \
+  --machine-type n1-standard-1 \
+  --metadata-from-file user-data=cloud-config/master-config.yaml
+```
+
+Deploy at least one worker node using the following command:
+```
+gcloud compute instances create \
+  kubernetes-node-1 \
+  --image-project coreos-cloud \
+  --image coreos-alpha-709-0-0-v20150611 \
+  --machine-type n1-standard-1 \
+  --metadata-from-file user-data=cloud-config/node-config.yaml
+```
+
+## 3. Using your cluster
+### 3.1 Configuring kubectl
+The following steps configure remote kubectl access to your cluster.
+
+Download `kubectl`
+```
+wget https://storage.googleapis.com/kubernetes-release/release/v1.1.3/bin/linux/amd64/kubectl
+```
+
+Get the IP address of your kubernetes-master
+```
+gcloud compute instances describe kubernetes-master | grep networkIP
```

Comment:
Hmm.. good point.  It should probably be secured, but didn't want to go through the trouble of TLS for this guide.  Maybe there is another way.